src/flows/pipes/RemoteCSV/view.tsx

```diff
@@ -0,0 +1,89 @@
+import React, {FC, useContext} from 'react'
+
+import {
+  Dropdown,
+  ComponentColor,
+  Input,
+  InputType,
+  ComponentStatus,
+} from '@influxdata/clockface'
+import {PipeProp} from 'src/types/flows'
+import {PipeContext} from 'src/flows/context/pipe'
+
+const SampleCSVs = {
+  Sample_Air_Sensor_Data:
+    'https://raw.githubusercontent.com/influxdata/influxdb2-sample-data/master/air-sensor-data/air-sensor-data-annotated.csv',
+  NOAA_Weather_Data: 'https://influx-testdata.s3.amazonaws.com/noaa.csv',
+  USGS_Earthquake_Data:
+    'https://raw.githubusercontent.com/influxdata/influxdb2-sample-data/master/usgs-earthquake-data/all_week-annotated.csv',
+  'NOAA_National_Buoy_Data_Center_(NDBC)':
+    'https://raw.githubusercontent.com/influxdata/influxdb2-sample-data/master/noaa-ndbc-data/latest-observations-annotated.csv',
+  Custom: '',
```

Comment:
Is there a way to get this as a source of truth from somewhere else?

Reply:
not that i know of

Reply:
bummer :(

Reply:
why?

Reply:
in the event that NOAA data gets removed or whatever. Like, what happens if we put an invalid URL?

Reply:
@ShmuelLotman are the underscores required?

Reply:
negative

Reply:
@russorat not technically, but object keys feel wrong without some sort of connector between the words

Reply:
in code, i agree, but when displayed to the user, i'd opt for pretty. since you're already using parentheses in there, can we replace the _ with spaces and wrap them in single quotes?